### ISSUE TITLE
Wrong template is being used when specifying the alias argument for template baking

### ISSUE THREAD

**Issue by ndm2** [cakephp/bake#395]
The help for the `Template` tasks states that the third argument would be used as the filename:

```
Arguments:

controller  Name of the controller views to bake. You can use
            Plugin.name as a shortcut for plugin baking.
            (optional)
action      Will bake a single action's file. core templates are (index,
            add, edit, view) (optional)
alias       Will bake the template in <action> but create the filename
            after <alias>. (optional)
```

So when passing the following arguments to bake:

    Template Articles list index

I'd expect a file named `index.ctp` to be generated from the `list.twig` template. The actual result however is that it will create a file named `index.ctp` using the `index.twig` template, it's basically as if one would only use the second argument to specify an action/template.

Looking at the source the expected behavior is impossible, as [**the second argument will never be used**](https://github.com/cakephp/bake/blob/1.5.6/src/Shell/Task/TemplateTask.php#L154-L156) in case the third argument is defined. 

Should bake behave the way I'm expecting it, or is my expectation wrong, and the help is wrong/misleading?

**Comment by markstory** [comment]
> Should bake behave the way I'm expecting it, or is my expectation wrong, and the help is wrong/misleading?

The help text implies that the behaviour you were expecting is correct. I don't think bake has done that for quite some time though. Currently the 3rd argument is pointless, and I am inclined to make it work like the help says.

**Comment by markstory** [comment]
Fixed by #396
